server/rest/src/main/java/org/infinispan/rest/resources/ClusterResource.java

```diff
@@ -137,4 +154,41 @@ private CompletionStage<List<NodeDistributionInfo>> clusterDistribution() {
                return Immutables.immutableListWrap(collection.toArray(new NodeDistributionInfo[0]));
             });
    }
+
+   private CompletionStage<RestResponse> handleRaftMembers(RestRequest request) {
+      RaftManager raftManager = raftManager();
+      if (!raftManager.isRaftAvailable()) {
+         return CompletableFuture.completedFuture(invocationHelper.noContentResponse(request));
+      }
+      boolean isPretty = isPretty(request);
+      return asJsonResponseFuture(invocationHelper.newResponse(request), Json.array(raftManager.raftMembers()), isPretty);
+   }
+
+   private CompletionStage<RestResponse> handleAddRaftMember(RestRequest request) {
+      RaftManager raftManager = raftManager();
+      if (!raftManager.isRaftAvailable()) {
+         return CompletableFuture.completedFuture(invocationHelper.notFoundResponse(request));
+      }
+      String raftId = request.variables().get(MEMBER_PARAMETER);
+      if (raftId == null) {
```

Comment:
This method should only be invoked when the variable exists. You don't need to check for `null`.